packages/ddex/publisher/scratch/parseDelivery.ts

```diff
@@ -168,18 +322,20 @@ export async function parseDdexXml(xmlUrl: string, xmlText: string) {
 
         problems: [],
         soundRecordings: [],
-        images: []
+        images: [],
+        deal: releaseDeals[ref]
       }
 
+      // resolve audius genre
       release.audiusGenre = resolveAudiusGenre(release.subGenre, release.genre)
       if (!release.audiusGenre) {
-        release.problems.push(`GenreMatchFailed`)
+        release.problems.push(`NoGenre`)
```

Comment:
Enums/union types would be nice for these

Reply:
Did added a `status` enum to releases table.